src/analytics/TraceEvent.tsx

```diff
@@ -1,12 +1,11 @@
 import React, { Children, cloneElement, isValidElement, memo, PropsWithChildren, SyntheticEvent } from 'react'
 
 import { sendAnalyticsEvent } from '.'
-import { BrowserEvent, EventName } from '../constants/primitives'
 import { ITraceContext, Trace, TraceContext } from './Trace'
 
 type TraceEventProps = {
-  events: BrowserEvent[]
```

Comment:
Should these be kept in the core package? These should not update based on new events being added and are more browser API primitives right?